install/kubernetes/helm/istio/values.yaml

```diff
@@ -372,7 +372,8 @@ pilot:
   autoscaleMax: 5
   image: pilot
   sidecar: true
-  traceSampling: 100.0
+  # Default value for production must be low, otherwise ingress may crash.
+  traceSampling: 1.0
```

Comment:
set to 100 for istio-demo.. else tasks wont work

Reply:
+1 on setting to 100 for demo :)